Makefile

```diff
@@ -5,12 +5,29 @@ default: help
 help:
 	@echo "make help              Show this help message"
 	@echo "make services          Run the services that Concourse and the pipeline requires"
+	@echo "make initdb            Restore DB dump on first run, can be also used to restore DB"
 	@echo "make sql               Connect to the cnx-db database with a psql shell"
 
+
+dump.sql.gz:
+	@echo Downloading db dump file first...
+	rsync -tv --progress -e 'ssh -T -c aes128-ctr -o Compression=no -x' backup1.cnx.org:/var/backups/db_dump/20190805_dump.sql.gz ./dump.sql.gz
+	@echo Please run "make initdb" to restore dump
+
+
 .PHONY: services
-services:
+services: dump.sql.gz
 	docker-compose up -d
 
+
+.PHONY: initdb
+initdb:
+	@echo drop and recreate db then restore dump
+	docker-compose exec -u postgres cnx-db dropdb -U postgres repository
+	docker-compose exec -u postgres cnx-db createdb -U postgres -O rhaptos repository
+	gunzip <dump.sql.gz | docker-compose exec -T -u postgres cnx-db psql -U postgres repository
```

Comment:
We'll likely need to verify ownership of the tables. I'd guess it's good though.

Reply:
The dump is made out of a cnx-db docker image so it should match name/ownership wise.